concent_api/api-integration-force-report-computed-task-test.py

```diff
@@ -75,15 +84,18 @@ def ack_report_computed_task(task_id, provider_private_key, provider_public_key,
 def main():
     cluster_url     = parse_command_line(sys.argv)
     task_id         = str(random.randrange(1, 100000))
+    subtask_id      = str(random.randrange(1, 100000))
```

Comment:
Try not get random value 2 times. Maybe better way to do it is to depend `task_id` on `subtask_id`? eg. ```task_id = subtask_id + 'x'```

Reply:
Done.